core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -572,7 +572,7 @@
     {{ include_js_file('expressions/expressionInterpolationService.js') }}
   </script>
 
+  <script src="{{ASSET_DIR_PREFIX}}/extensions/interactions/baseValidator.js"></script>
```

Comment:
Ah I see. Initially I was confused, but now I see this is needed because it's under the core/ path. This is fine; thanks!